requirements/edx/base.in

```diff
@@ -56,7 +56,7 @@ django-ratelimit
 django-ratelimit-backend==1.1.1
 django-require
 django-sekizai
-django-ses==0.8.4
+django-ses==0.8.13
```

Comment:
Please go ahead and remove the version constraint, so we can keep it up to date more easily via `make upgrade`.